src/transformer/utils/get_attempt_result.php

```diff
@@ -17,15 +17,17 @@
 namespace src\transformer\utils;
 defined('MOODLE_INTERNAL') || die();
 
-function get_attempt_result(array $config, $attempt, $gradeitem) {
+use src\transformer\utils as utils;
+
+function get_attempt_result(array $config, $attempt, $grade_item) {
     $gradesum = isset($attempt->sumgrades) ? $attempt->sumgrades : 0;
 
-    $minscore = floatval($gradeitem->grademin ?: 0);
-    $maxscore = floatval($gradeitem->grademax ?: 0);
-    $passscore = floatval($gradeitem->gradepass ?: 0);
+    $minscore = (float)($grade_item->grademin ? $grade_item->grademin : 0);
+    $maxscore = (float)($grade_item->grademax ? $grade_item->grademax : 0);
+    $passscore = (float)($grade_item->gradepass ? $grade_item->gradepass : 0);
 
-    $rawscore = cap_raw_score($gradesum, $minscore, $maxscore);
-    $scaledscore = get_scaled_score($rawscore, $minscore, $maxscore);
+    $rawscore = floatval((cap_raw_score($gradesum, $minscore, $maxscore)));
```

Comment:
I think we should probably just use `floatval` in the `cap_raw_score` and `get_scaled_score` functions to make this a bit cleaner to read if you agree @caperneoignis

Reply:
Yea, I will see if that works. I think there is a reason I was not doing that, let me try it and see if the issue is resolved. I think it has something to do with the function that is being called. I think that is setting it differently.... But I don't remember which means I should do it again, and verify my findings.